modules/system/controllers/Updates.php

```diff
@@ -181,17 +181,27 @@ protected function getPluginVersionFile($path, $filename)
         $contents = [];
 
         try {
-            $updates = Yaml::parseFile($path.'/'.$filename);
-            $updates = is_array($updates) ? array_reverse($updates) : [];
+            $updates = (array)Yaml::parseFile($path.'/'.$filename);
```

Comment:
space between `(array)` and `Yaml` please.

Reply:
Done